court_ball_game/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from court.models import Court
+from player.models import BallGame
+
+
+class CourtBallGame(models.Model):
+    court_id = models.ForeignKey(Court, on_delete=models.CASCADE)
```

Comment:
Looking at the tests, I think this should be named court rather than court_id, as it contains a reference to the court rather than its id
```suggestion
    court = models.ForeignKey(Court, on_delete=models.CASCADE)
```